config/webpack.config.js

```diff
@@ -46,6 +46,10 @@ module.exports = {
                     ]
                 },
             },
+            {
+                test: /\.css$/,
+                loader: 'style-loader!css-loader?modules=true&localIdentName=[name]__[local]___[hash:base64:5]'
```

Comment:
If you are using `webpack@2`, you can do:

```
{
 test: /\.css$/,
  use: [
    { 
      loader:'style-laoder
    }, 
    {
      loaders: 'css-loader',
      options: {
        modules: true,
        localIndentName: '[name]__[local]__[hash:base64:5]'
      }
    }
  ]
}
```

I think it makes it easier to read.